interpreter/apmint/apmint.go

```diff
@@ -153,7 +153,11 @@ func hashTrace(trace *libpf.Trace) (hash [16]byte) {
 	h := fnv.New128a()
 	for _, uniqueFrame := range trace.Frames {
 		frame := uniqueFrame.Value()
-		_, _ = h.Write(frame.FileID.Bytes())
+		fileID := libpf.FileID{}
+		if frame.MappingFile != (libpf.FrameMappingFile{}) {
```

Comment:
nit: It would be great to have a helper, that checks if a given libpf.FrameMappingFile{} does not hold a value. From my perspective, it would improve code readability.

Reply:
I think we'd then need to make `libpf.FrameMappingFile` a real type instead of type alias, and a bit of additional glue similar to how `libpf.String` was done. Would this be the prefered pattern?